SECURITY.md

```diff
@@ -1,11 +1,8 @@
-We acknowledge that every line of code that we write may potentially contain
-security issues.
+We acknowledge that every line of code that we write may potentially contain security issues.
+We are trying to deal with it responsibly and provide patches as quickly as possible.
 
-We are trying to deal with it responsibly and provide patches as quickly as
-possible. If you have anything to report to us please use the following
-channels:
+We host our bug bounty program on HackerOne, it is currently private, therefore if you would like to report a vulnerability and get rewarded for it, please ask to join our program by filling this form:
 
-Email: Tech-Security@zalando.de
-OR
-Submit your vulnerability report through our bug bounty program at:
-https://hackerone.com/zalando
+https://corporate.zalando.com/en/services-and-contact#security-form
+
+You can also send you report via this form if you do not want to join our bug bounty program and just want to report a vulnerability or security issue.
```

Comment:
typo: "you" -> "your" (maybe the template is wrong?)

Reply:
yes template is wrong :'(